receiver/filelogreceiver/filelog_test.go

```diff
@@ -167,15 +173,6 @@ func (rt *rotationTest) Run(t *testing.T) {
 	fileName := filepath.Join(tempDir, "test.log")
 	backupFileName := filepath.Join(tempDir, "test-backup.log")
 
-	// Build expected outputs
```

Comment:
the generated entries for the expected log entries have not been used previously, as the `require.Equal` below has been commented out. I tried to make use of the `plogtest.CompareLogs` function to add the validation of the received log entries against the expected entries but ran into some troubles handling the order of the received log entries, which seems to be linked to the log entries being received in a random order during the log file rotation.
This however might be worth looking into as part of a separate PR dedicated to that.